src/facade/memcache_parser.cc

```diff
@@ -38,6 +38,15 @@ MP::CmdType From(string_view token) {
 
 using TokensView = absl::Span<std::string_view>;
 
+bool TokenToExpireTs(std::string_view token, std::uint32_t* expire_ts) {
+  static constexpr uint32_t month = 60 * 60 * 24 * 30;
+  auto res = absl::SimpleAtoi(token, expire_ts);
+  if (res && (*expire_ts <= month)) {
+    *expire_ts += time(nullptr);
```

Comment:
if token is beyond the limit, do we you still want to return success?
seems like a bug because you will return a relative number and return success

Reply:
I also do not understand why to add current time in the parser. why not keep the relative time and make the adjustment at later time during the command processing.

Reply:
He is converting it to unix time since epoch if it's less than a month in seconds. See my other comment